ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -51,13 +51,16 @@ Item {
             model: root.collectiblesModel
             cellHeight: 229
             cellWidth: 176
-            delegate: CollectibleView {
+            delegate: CollectibleView {               
                 height: gridView.cellHeight
                 width: gridView.cellWidth
-                collectibleModel: model
-                onCollectibleClicked: {
-                    root.collectibleClicked(address, tokenId);
-                }
+                title: model.name ? model.name : "..."
+                subTitle: model.collectionName ? model.collectionName : ""
+                imageUrl: model.imageUrl ? model.imageUrl : ""
+                backgroundColor: model.backgroundColor ? model.backgroundColor : "transparent"
+                isLoading: root.isLoading
```

Comment:
That property does not exist in `root`, it's in `model`.

Reply:
True! Thanks!